### DISCUSSION TITLE
osgEarth::CompositeImageLayer can not show correctly when the input layers have projected coordinate system

### DISCUSSION THREAD

**Discussion by chenguanga** [pelicanmapping/osgearth#1975] [Support Q&A]
In the osgearth_map example, when I changed the input layers with  projected coordinate system(WGS_1984_Transverse_Mercator), osgEarth::CompositeImageLayer can only  show the first layer, the second one is disappeared，such as below:
![QQ截图20220213185443](https://user-images.githubusercontent.com/36613682/153749909-37b94793-a886-4944-9a36-c7be9859620a.jpg)
When I convert the above two layers to geographic coordinate system(WGS_1984), The results are displayed correctly, such as below:
![QQ截图20220213185817](https://user-images.githubusercontent.com/36613682/153750034-a4c9daa6-b693-455e-bcec-8bf0223457aa.jpg)
When I load the two layers seperately by using osgEarth::GDALImageLayer, no matter what coordinate system is used, the result is correct. How can I solve this problem?

**Comment by gwaldron** [comment]
Can you reproduce the problem using an earth file? If so, please attach it and we will take a look.

**Reply by chenguanga** [reply]
[composite_image.zip](https://github.com/gwaldron/osgearth/files/8064995/composite_image.zip)
This file contains two folders(Geographic and Projected), each has an earth file that can reproduce the problem(osgEarth version: Master).
